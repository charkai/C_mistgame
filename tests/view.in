place B2
view
place C3
view
place M2
view
view
view
term
