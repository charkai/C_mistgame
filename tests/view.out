L17,xxxxxxx..........................................
B1,xx.....xx..o..xx.#...xx.....xxxxxxxxxxxxxxxxxxxxx
O17,xxxxxxx..........................................
O17,xxxxxxx..........................................
O17,xxxxxxx..........................................
